src/chainstate/stacks/boot/pox-2.clar

```diff
@@ -866,14 +889,22 @@
 
       ;; register the PoX address with the amount stacked
       ;;   for the new cycles
-      (let ((reward-set-indexes (try! (add-pox-addr-to-reward-cycles pox-addr first-extend-cycle extend-count amount-ustx tx-sender))))
+      (let ((extended-reward-set-indexes (try! (add-pox-addr-to-reward-cycles pox-addr first-extend-cycle extend-count amount-ustx tx-sender)))
+            (reward-set-indexes (match stacker-state
+                                       ;; if there's active stacker state, we need to extend the existing reward-set-indexes
+                                       old-state (let ((cur-cycle-index (- first-reward-cycle (get first-reward-cycle old-state)))
+                                                       (old-indexes (get reward-set-indexes old-state))
+                                                       (new-list (concat (default-to (list) (slice old-indexes cur-cycle-index (len old-indexes)))
+                                                                                   extended-reward-set-indexes)))
+                                            (unwrap-panic (as-max-len? new-list u12)))
+                                       extended-reward-set-indexes)))
           ;; update stacker record
           (map-set stacking-state
             { stacker: tx-sender }
-            { pox-addr: pox-addr,
+            (print { pox-addr: pox-addr,
```

Comment:
Is the `print` going to be dropped before this merges?

Reply:
Yes

Reply:
Dropped in [9ca085b](https://github.com/stacks-network/stacks-blockchain/pull/3299/commits/9ca085bccfb1973704d50f2fd58c5a26a16bcac2)